### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by freliciatucker**

### CODE REVIEW

**Comment by D-moe** [comment]
@googlebot I fixed it.

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  <build>
+    <plugins>
+      <!-- Provides `mvn package appengine:run` for local testing
+           and `mvn package appengine:deploy` for deploying. -->
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>appengine-maven-plugin</artifactId>
+        <version>2.2.0</version>
+        <configuration>
+         <!-- TODO: Set project ID. -->
```
Comment (zjwoody) [inline pom.xml:41]: You can set capstone project id: pose-3d-viewer-step-2020

File: target/portfolio-1/index.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>My first three.js app</title>
+        <link rel="stylesheet" href="style.css">
+	</head>
+	<body>
+        <h1>Capstone project page</h1>
```
Comment (zjwoody) [inline target/portfolio-1/index.html:9]: Give a meaningful project name, e.g. Pose 3D Viewer

File: target/portfolio-1/index.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
```
Comment (zjwoody) [inline target/portfolio-1/index.html:4]: Have you run css/html/js format check to clean the format?
Try to add an automatic  format check in your github repo to make sure every pr is clean.

File: target/maven-archiver/pom.properties [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
+#Tue Jul 21 14:32:08 UTC 2020
+version=1
+groupId=com.google.sps
+artifactId=portfolio
```
Comment (zjwoody) [inline target/maven-archiver/pom.properties:5]: Why the id is "portfolio"?

File: pom.xml (hunk 1 above)
Comment (freliciatucker) [reply pom.xml:41]: set in deploy branch

File: target/maven-archiver/pom.properties (hunk 4 above)
Comment (freliciatucker) [reply target/maven-archiver/pom.properties:5]: I changed it to  Pose 3D Viewer

File: target/portfolio-1/index.html (hunk 3 above)
Comment (freliciatucker) [reply target/portfolio-1/index.html:4]: I'm looking on the action tab to see how to add the format checks and the automatic deploy whenever a branch is merged with master. If I'm still having trouble finding it, I will definitely reach out to the chat!

**Pull request closed without merging**